lib/shopify_api/api_version.rb

```diff
@@ -0,0 +1,87 @@
+module ShopifyAPI
+  class ApiVersion
+    class UnknownVersion < StandardError; end
+
+    class NoVersion < ApiVersion
+      API_PREFIX = '/admin/'.freeze
+
+      def initialize
+        @version_name = "no_version"
+      end
+
+      def construct_api_path(path)
+        "#{API_PREFIX}#{path}"
+      end
+
+      def construct_graphql_path
+        '/admin/api/graphql.json'
```

Comment:
Re-use `API_PREFIX`?

Reply:
Or just use `construct_api_path`?

Reply:
i'll move this into a constant,  i don't think sharing the API prefix is worth it in this case.